modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+
+[id="minimum-required-azure-permissions_{context}"]
+= Required Azure permissions in installer-provisioned infrastructure
+
+When you assign the `Contributor` and `User Access Administrator` roles to the service principal, you automatically grant all the required permissions. To deploy and destroy an {product-title} cluster with installer-provisioned infrastructure, the service principal requires the following permissions.
+
+Following are the permissions required for creating the {product-title} cluster.
```

Comment:
Do we need to mention that those permissions are only applicable for azure Public Cloud?  As this doc is also used to create account for Azure GovCloud, but we only verify them on Azure Public Cloud. @chiragkyal WDYT?

Reply:
Interesting! yeah we can mention that it is only applicable for Azure public cloud. So it would be like :

`Following are the permissions required for creating the {product-title} cluster in Azure Public Cloud`, right ?

Reply:
It looks good to me.

Reply:
Addressed

Reply:
Addressed